src/ui/screens/onboarding/OnboardingStartSlideMobile.qml

```diff
@@ -39,15 +40,21 @@ ColumnLayout {
         color: MZTheme.theme.fontColor
     }
 
+    Item {
+        Layout.fillHeight: true
+        Layout.minimumHeight: 16
```

Comment:
Is it worth putting constants like this (16) in theme.js?

Reply:
sure, I think this particular value for minimum spacing vertically around images in onboarding warrants a variable